src/bilder/components/docker/docker_compose_steps.py

```diff
@@ -17,6 +17,10 @@ def download_docker_compose():
         name="Make Executable",
         commands=["chmod +x /usr/local/bin/docker-compose"],
     )
+    server.shell(
```

Comment:
For this one I would use `files.directory` as the operation so that you can more easily manage the appropriate user/group ownership

Reply:
hmm can I just do a chmod in that case?